### PR TITLE
fix: pluralize of build and others

### PR DESCRIPTION
**Pull request by caarlos0**
This updates the flect depencency.

Doing it because I found that `build` is now wrongly pluralized to `buildren`.

This has been fixed on v0.2.1.

reproducible:

```go
package main

import "fmt"
import "github.com/gobuffalo/flect"

func main() {
    fmt.Println(flect.Pluralize("build"))
}
```

Test it with both v0.2.0 and v0.2.1, output will be `buildren` and `builds`, respectively.

refs https://github.com/gobuffalo/flect/releases/tag/v0.2.1

### CODE REVIEW

File: testdata/project-v2-addon/go.mod [hunk 1]
```diff
@@ -9,5 +9,5 @@ require (
 	k8s.io/apimachinery v0.17.2
 	k8s.io/client-go v0.17.2
 	sigs.k8s.io/controller-runtime v0.5.0
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200317144824-bbf1fb2a4a9a
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200413154948-0284e6ed8028
```
Comment (caarlos0) [inline testdata/project-v2-addon/go.mod:12]: not sure why this changed, just did it because tests was failing

Comment (camilamacedo86) [reply testdata/project-v2-addon/go.mod:12]: It was not because of your PR. Could you please rebase it with the master?

Comment (camilamacedo86) [reply testdata/project-v2-addon/go.mod:12]: Also, could we not add a test for its scenario in https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/model/resource/options_test.go? WDYT?

Comment (caarlos0) [reply testdata/project-v2-addon/go.mod]: merged!



> Also, could we not add a test for its scenario in https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/model/resource/options_test.go? WDYT?

not sure I get it... you meant, add a test to `builds` pluralize thing?

**Comment by caarlos0** [comment]
oh, seems like it was fixed on master already 

https://github.com/kubernetes-sigs/kubebuilder/commit/898536931153319f92cd37ee68f8c5e741222ef9#diff-37aff102a57d3d7b797f152915a6dc16R6

closing

thanks!

**Pull request closed without merging**